### DISCUSSION TITLE
Implement X-Up-Size

### DISCUSSION THREAD

**Discussion by leevigraham** [unpoly/unpoly#362] [Q&A]
In my use case:

* User request url that requires auth with `up-layer="new" up-size="large"`
* Login form is returned which is generally rendered with `up-layer="new" up-size="small"`
* Login for is rendered in `large` modal.

It would be great if a response header could change the size of the modal.

**Comment by triskweline** [comment]
This is also something you can hack yourself:

```js
up.on('up:fragment:loaded', function(event) {
  let serverSize = event.response.getHeader('App-Size')
  if (serverSize) {
    event.renderOptions.size = serverSize
  }
})
```
